docs/DeveloperGuide.md

```diff
@@ -739,101 +638,80 @@ otherwise)
 
 ****
 
+**Use case: Delete question**
 
-**Use case: Mark question**
-
-1. User requests to list questions
-2. SETA shows list of questions
-3. User requests to mark a specific question as important
-4. SETA marks the question as important
+1. User requests to delete a specific question in the list.
+2. SETA deletes the question.
 
    Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 3a. The given index is invalid.
+* 2a. The given index is invalid.
```

Comment:
I think this comes before 2 so we can make it 1b instead?

Reply:
resolved